src/dev/build/tasks/os_packages/docker_generator/templates/base/Dockerfile

```diff
@@ -115,7 +114,7 @@ RUN for iter in {1..10}; do \
     (exit $exit_code)
 {{/ubuntu}}
 {{#chainguard}}
-RUN apk --no-cache add bash curl fontconfig libstdc++ freetype nss findutils shadow
+RUN apk --no-cache add bash fontconfig libstdc++ nss findutils shadow
```

Comment:
unrelated cleanup
- curl isn't needed in this stage.
- freetype isn't needed, there's a noto font download in stage 0.